stix_shifter/stix_translation/src/modules/guardium/query_constructor.py

```diff
@@ -0,0 +1,524 @@
+from stix_shifter.stix_translation.src.patterns.pattern_objects import ObservationExpression, ComparisonExpression, \
+    ComparisonExpressionOperators, ComparisonComparators, Pattern, \
+    CombinedComparisonExpression, CombinedObservationExpression, ObservationOperators
+from stix_shifter.stix_translation.src.utils.transformers import TimestampToMilliseconds
+from stix_shifter.stix_translation.src.utils.transformers import TimestampToGuardium
+from stix_shifter.stix_translation.src.json_to_stix import observable
+from stix_shifter.stix_translation.src.utils.stix_pattern_parser import parse_stix
```

Comment:
We no longer return the `parsed_stix` as part of the query translation so this can be removed.